themes/default/files/scripts/exam-display.js

```diff
@@ -489,7 +489,9 @@ Numbas.queueScript('exam-display',['display-base','math','util','timing'],functi
                     this.startTime(exam.start);
                     this.endTime(exam.stop);
                     break;
-                case "suspend":
+                case "paused":
+                    break;
```

Comment:
if you get rid of this `break` line, then both 'paused' and 'resumed' will lead to the following block.